ui/src/assets/images/guardian-logo-320.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="320" height="60">
```

Comment:
It's enabled here https://github.com/guardian/frontend/pull/17539/files#diff-0e5570e40024e4d680d2c1d8a4adedfdR11